src/app/[prefecture]/all/page.tsx

```
@@ -0,0 +1,230 @@
+        overlays
+      ).addTo(mapInstance);
+
+      // Set initial view based on block parameter
+      const setInitialView = () => {
+        let latlong: [number, number];
+        let zoom: number;
+        
+        if (block && prefectureConfig.blocks) {
+          const blockConfig = prefectureConfig.blocks.find(b => b.id === block);
+          if (blockConfig && blockConfig.lat && blockConfig.long && blockConfig.zoom) {
+            latlong = [blockConfig.lat, blockConfig.long];
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)